examples/helpers/mail/mail_example.py

```diff
@@ -101,6 +101,36 @@ def build_attachment2():
     attachment.content_id = "Banner"
     return attachment
 
+def build_attachment3_s3_store_default_session():
+    """
+    Build attachment mock to be stored in s3
+    with default session.
+    Read Boto docs to setup default AWS config
+    """
+    attachment = S3Attachment()
+    attachment.content = "bG9yZW1pcHN1bWZvb2JhcnNk"
+    attachment.type = "application/pdf"
+    attachment.filename = "sample.pdf"
+    attachment.disposition = "attachment"
+    attachment.content_id = "S3AttachmentSample"
+    return attachment
+
+def build_attachment4_s3_store_custom_session():
+    """ Build attachment mock to be stored in s3 with custom session."""
+    attachment = S3Attachment()
+    attachment.content = "bG9yZW1pcHN1bWZvb2JhcnNk"
+    attachment.type = "application/pdf"
+    attachment.filename = "foo.pdf"
+    attachment.disposition = "attachment"
+    attachment.content_id = "S3Session"
+    attachment.bucket = YOUR_BUCKET_NAME
+    attachment.session =    (aws_access_key_id=YOUR_ACCESS_KEY,
+                            aws_secret_access_key=YOUR_SECRET_KEY,
+                            aws_session_token=SESSION_TOKEN,
+                            region_name=REGION_NAME)
```

Comment:
I don't think `attachment.session` is going to work. The indentation is what made me take a second look but then I realized it is invalid syntax as is. Is this supposed to be a function call or maybe assigning a dict to `attachment.session` like this?

```python
    attachment.session = dict(
        aws_access_key_id=YOUR_ACCESS_KEY,
        aws_secret_access_key=YOUR_SECRET_KEY,
        aws_session_token=SESSION_TOKEN,
        region_name=REGION_NAME
    )
```